recipes/drwav/all/CMakeLists.txt

```diff
@@ -1,34 +1,37 @@
 cmake_minimum_required(VERSION 3.4)
-project(dr_wav C)
+project(dr_wav LANGUAGES C)
 
-include(conanbuildinfo.cmake)
-conan_basic_setup(KEEP_RPATHS)
+include(GNUInstallDirs)
 
 option(NO_CONVERSION_API "Build with define DR_WAV_NO_CONVERSION_API" OFF)
 option(NO_STDIO "Build with define DR_WAV_NO_STDIO" OFF)
+option(NO_WCHAR "Build with define DR_WAV_NO_WCHAR" OFF)
 
-add_library(${CMAKE_PROJECT_NAME} dr_wav.c)
```

Comment:
Alright, reverted in 077c8ed782dc126b590a6c53c2e0dff043e087f1. I changed it because I saw it changed like that in other recipes that were updated to v2.